docs/examples/interpolated-periods.py

```diff
@@ -0,0 +1,68 @@
+"""
+Interpolated Data Periods
+=========================
+
+Identifying periods in a time series where the data has been
+linearly interpolated.
+"""
+
+# %%
+# Identifying periods where time series data has been linearly interpolated
+# and removing these periods may help to reduce noise when performing future
+# data analysis. This example shows how to use
+# :py:func:`pvanalytics.quality.gaps.interpolation_diff`, which identifies and
+# masks linearly interpolated periods.
+
+import pvanalytics
+from pvanalytics.quality import gaps
+import matplotlib.pyplot as plt
+import pandas as pd
+import pathlib
+
+# %%
+# First, we import the AC power data stream that we are going to check for
+# interpolated periods. The time series we download is a normalized AC power
+# time series from the PV Fleets Initiative, and is available via the DuraMAT
+# DataHub:
+# https://datahub.duramat.org/dataset/inverter-clipping-ml-training-set-real-data
```

Comment:
Similar comment as above - add comments describing data requirements, e.g., is it assumed (or required) that the input AC power is serially complete? In this example, it would help to point out that there are gaps in the data which will be filled in by interpolation, for the purpose of creating a dataset with linear features to illustrate their identification.  Otherwise, it may be confusing why we are interpolating in order to detect interpolations.